### PR TITLE
Fix: Remove unnecessary log spam

### LINKED ISSUE #11918: Code Quality: `DetectQuickLookAvailability|QuickLook detected: False` spammed in log file
### Description

No issue with the app in general, just log spam. A new entry is added when navigating to another folder

### Steps To Reproduce

1. Open log file
2. Browe a folder
3. See new entries

### Requirements

- Stop logging if QuickAccess was detected

### Files Version

2.4.59.0

### Windows Version

10.0.22621.1413

### Log file

[debug.log](https://github.com/files-community/Files/files/11121079/debug.log)

### PR DESCRIPTION
**Pull request by hecksmosis**
**Resolved / Related Issues**
- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Closes #11918

**Validation**
How did you test these changes?
- [x] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [x] Are there any other steps that were used to validate these changes?
   1. Open app
   2. See it still works

**Screenshots (optional)**
Add screenshots here.

### CODE REVIEW

File: src/Files.App/Helpers/QuickLookHelpers.cs [hunk 1]
```diff
@@ -64,7 +64,6 @@ static async Task<int> QuickLookServerAvailable()
 		try
 		{
 			var result = await QuickLookServerAvailable();
-			App.Logger.Info($"QuickLook detected: {result != 0}");
```
Comment (hishitetsu) [inline src/Files.App/Helpers/QuickLookHelpers.cs:67]: I think it should output a log if QuickLook not detected when LaunchQuickLookAction is executed.

Comment (yaira2) [reply src/Files.App/Helpers/QuickLookHelpers.cs:67]: 👍

**Comment by lukeblevins** [comment]
Hey @hecksmosis,

I went ahead and incorporated the logging changes from this PR into my branch that has larger changes here: #11937

**Review by lukeblevins — COMMENTED** [review]
See comment below 👍🏻

**Comment by hecksmosis** [comment]
Ok, so should I close this?

**Comment by yaira2** [comment]
> Ok, so should I close this?

We can keep it open, I want to include this in the next servicing release.

**Review by yaira2 — APPROVED** [review]
LGTM

**Pull request merged**